code/api/src/modules/user/mutations.test.js

```diff
@@ -0,0 +1,48 @@
+import request from 'supertest'
+import express from 'express'
+import graphqlHTTP from 'express-graphql'
+import schema from '../../setup/schema'
+import models from '../../setup/models'
+import bcrypt from 'bcrypt';
+import db from '../../setup/database';
+
+describe('user mutations',  () => {
+  let server;
+
+  beforeAll(() => {
+    server = express();
+
+    server.use(
+      '/',
+      graphqlHTTP({
+        schema: schema,
+        graphiql: false,
+      })
+    )
+  })
+
+ afterEach(async() => {
+   await models.User.destroy({ where: {}})
+ })
+
+ afterAll(() => {
+   db.close()
+ })
```

Comment:
I believe this wipes the database after each test. Jest is different from Rails in that creating users in one test effects the other tests running concurrently. Here we are adding a user through a mutation and at the end of the test suite returning to a blank slate.

Reply:
Nice Work!

Reply:
Very cool use of afterAll() hook